plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -831,6 +837,52 @@ public void testSelectNestedTimestamps()
         assertQuery("SELECT CAST(col1[1].ts AS VARCHAR) FROM nested_timestamps_parquet_stats LIMIT 1", "VALUES '2010-02-03 12:11:10.000 UTC'");
     }
 
+    @Test
+    public void testConvertJsonStatisticsToParquetOnRowType()
+            throws Exception
+    {
+        verifySupportsInsert();
```

Comment:
a "verify ..." should verify, i.e. ensure something is true

as a follow-up we could rename this to eg `skipUnlessInsertsSupported`

Reply:
I will send a follow-up PR.